### ISSUE TITLE
AR sessions opens and closes immediately on using blob URL as flie in AR Intent

### ISSUE THREAD

**Issue by optimus007** [google-ar/arcore-android-sdk#1297]
### SPECIFIC ISSUE ENCOUNTERED

after making a glb or gltf file using three js gltfExporter 

and the resulting buffer is converted to blob using `blob = new Blob([buffer], { type: 'application/octet-stream' })`

then   URL is made using  `let blobUrl = URL.createObjectURL(blob)`

and finally the intent URL
``` javascript
   link.href = `intent://arvr.google.com/scene-viewer/1.0?file=${blobUrl}#Intent;scheme=https;package=com.google.android.googlequicksearchbox;action=android.intent.action.VIEW;S.browser_fallback_url=https://developers.google.com/ar;end;`
   ```
example
``` javascript
  intent://arvr.google.com/scene-viewer/1.1?file=blob%3Ahttp%3A%2F%2F127.0.0.1%3A5500%2F66b63935-7600-42cb-b54e-85bb4f8094d8&mode=ar_only&link=https%3A%2F%2Fwww.google.com%2F#Intent;scheme=https;package=com.google.ar.core;action=android.intent.action.VIEW;S.browser_fallback_url=https://www.google.com/;end
```
clicking this opens and closes the AR session immediately without any errors

**live example :**
click the `start AR` button on chrome on mobile
`export file` for the glb or gltf file 
`example AR` to start a ar session from glb url
https://codepen.io/optimus007-the-looper/pen/QWMwKbB

### VERSIONS USED

- Device manufacturer, model, and O/S: OnePlus Nord 2,android 11
- Google Play Services for AR (ARCore):  1.27.212430183

**Comment by pawel-marciniak** [comment]
Hi @optimus007 , did you find some solution for this? Is AR explicit intent even supports Blob?

**Comment by optimus007** [comment]
> Hi @optimus007 , did you find some solution for this? Is AR explicit intent even supports Blob?

Nope...  Only works with real URLs

You could use webXR but it's bare bones and not as stable

**Comment by pawel-marciniak** [comment]
@optimus007 ohh, it's a shame, QuickLook on iOS works with Blobs :/ Thanks for answer anyway :)

**Comment by optimus007** [comment]
> @optimus007 ohh, it's a shame, QuickLook on iOS works with Blobs :/ Thanks for answer anyway :)

Exactly... Google is pretty bad at staying up to date with...  Hope khronos group can take over all ar projects

Gltf format has become quite amazing

**Comment by Patrick-van-Halm-360Fabriek** [comment]
Will there ever be support for ObjectURL's and Blobs within ARCore?

**Comment by florian-forestier** [comment]
Hey, thanks for the feedback 😄 

I cannot use a remote storage (for various security concerns), and I'm already using WebXR when available. I'm using ARCore only as a fallback (ie. for users on Firefox Android). As a workaround, I simply download the file and asks the user to open it. Not as straightforward as it should, but still not so bad.
